src/content/protocols/pancakeswap-v2-bsc.md

```diff
@@ -0,0 +1,150 @@
+| MasterChefV2   | updateCakeRate    | Updates the percentages sent to burn, regular, and special farms. This alters CAKE’s inflation schedule. Setting burn rate to 0 % while doubling farm rewards would inflate supply rapidly and erode token value.                                                                         | Timelock `0xA1f482Dc…e8fE4`     |
+| MasterChefV2   | burnCake          | Transfers accumulated CAKE to the `burnAdmin` address. Intended to reduce supply, it actually sends tokens to whatever address is set. If that address is switched to an attacker wallet, “burns” become free payouts.                                                                    | Timelock `0xA1f482Dc…e8fE4`     |
+| MasterChefV2   | transferOwnership | Writes a new `owner` for MasterChefV2. Control over emissions, pool lists, and admin settings moves with it. Transferring to a rogue EOA would let a single key redirect or freeze all farming rewards.                                                                                   | Timelock `0xA1f482Dc…e8fE4`     |
+| Timelock       | setDelay          | Changes the global execution delay for queued governance calls. This defines the minimum exit window for users. Dropping the delay to the 6-hour minimum gives LPs little time to react to hostile proposals.                                                                             | Timelock itself                 |
+| CakeToken      | mint              | Calls `_mint`, increasing total CAKE supply and sending tokens to any address. This is the root of all farming emissions. MasterChef V1 is the only contract that can call this mint function, it is a fixed 40 CAKE per block mint maximum int MasterChef's logic.                       | MasterChef V1 `0x73feaa1e…e24E` |
+| CakeToken      | transferOwnership | Assigns a new owner with full mint authority. Control of CAKE inflation moves entirely to the new address. If passed to a malicious contract, unlimited CAKE could be minted, destroying token economics. It is uncallable function since MasterChef V1 functions have no way to call it. | MasterChef V1 `0x73feaa1e…e24E` |
+
+## Dependencies
+
+| Dependency          | Type     | Risk if it fails                                                       |
+| ------------------- | -------- | ---------------------------------------------------------------------- |
+| Binance Smart Chain | L1 chain | All swaps & CAKE emission halt                                         |
+| CAKE token          | ERC-20   | Reward flow stops                                                      |
+| GnosisSafe multisig | Admin    | Parameter changes blocked, protocol fee's and CAKE emissions hijacked, |
```

Comment:
Please also use full sentences in this section. The goal here is to go in more technical details than in ## Autonomy